src/sol_imgui/sol_imgui.h

```diff
@@ -2,6 +2,13 @@
 
 namespace sol_ImGui
```

Comment:
also, please, document in sol_ImGui readme present in the same folder, it is something i sort of care about as i tried to keep that file as an updated reference of sorts

Reply:
*document the new overloads